site/data/kapp-controller/docs/kapp-controller-develop-toc.yml

```diff
@@ -27,6 +27,10 @@ toc:
         url: /packaging-tutorial
       - page: "Consuming Packages using the CLI"
         url: /kctrl-package-tutorial
+      - page: Packaging upstream artifacts with CLI
+        url: /kctrl-package-authoring-of-upstream
+      - page: Building packages from source using kctrl
```

Comment:
nit: one page says: `Packaging upstream artifacts with CLI` and other page says: `Building packages from source using kctrl`. 
May be we can make them consistent using `Packaging upstream artifacts using the CLI` and `Packaging from source code using the CLI`

Reply:
Not relevant anymore because we are merging the two